### ISSUE TITLE
Show two (or more) images

### ISSUE THREAD

**Issue by mrAceT** [mistic100/Photo-Sphere-Viewer#428]
I took a look at:
https://photo-sphere-viewer.js.org/guide/cropped-panorama.html#theory

Nice! I do have a question, is it possible to overlay an image this way, over an existing 360 pano (equirectangular or cubical) or combine several "standard" images ?

**Comment by mistic100** [comment]
No. I think this is beyond the scope of the library.  
You can stich your multiple images yourself using a canvas and provide de base64 image data for the "panorama" parameter.

**Comment by mrAceT** [comment]
A pitty, but could it be possible then to make the rest of the canvas transparant around that image?

I assume it is possible to run multiple instances of PSV in one page, then one could overlay one with the other.. and via javascript copy the motions onto the other..

**Comment by mistic100** [comment]
If it was simple I would have considered it, it is not. The viewer uses a sphere geometry which can display only one texture. What you ask requires deep modifications to display multiple spheres.

TLDR : I don't want to implement this level of complexity/customisation.
